src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts

```diff
@@ -276,17 +276,20 @@ export class TransactionsInProgressTableDataSource extends TableDataSource<Trans
 
   public buildTableFiltersDef(): TableFilterDef[] {
     const filters: TableFilterDef[] = [];
+    const issuerFilter = new IssuerFilter().getFilterDef();
     // Show Site Area Filter If Organization component is active
     if (this.componentService.isActive(TenantComponents.ORGANIZATION)) {
-      filters.push(new IssuerFilter().getFilterDef());
-      filters.push(new SiteTableFilter().getFilterDef());
-      filters.push(new SiteAreaTableFilter().getFilterDef());
+      const siteFilter = new SiteTableFilter(null, [issuerFilter]).getFilterDef();
+      filters.push(issuerFilter);
+      filters.push(siteFilter);
+      filters.push(new SiteAreaTableFilter([siteFilter, issuerFilter]).getFilterDef());
     }
     filters.push(new ChargingStationTableFilter().getFilterDef());
     filters.push(new ConnectorTableFilter().getFilterDef());
-    if (this.authorizationService.isAdmin() || this.authorizationService.hasSitesAdminRights()) {
-      filters.push(new UserTableFilter(this.authorizationService.getSitesAdmin()).getFilterDef());
-      filters.push(new TagTableFilter().getFilterDef());
+    if ((this.authorizationService.isAdmin() || this.authorizationService.hasSitesAdminRights())) {
+      filters.push(new UserTableFilter(this.authorizationService.getSitesAdmin(),
+        this.componentService.isActive(TenantComponents.ORGANIZATION) ? [issuerFilter] : []).getFilterDef());
```

Comment:
issuerFilter has nothing to do with Organization check I think which can be removed.

Reply:
Done